### PR TITLE
[nit] integration-cli: obey Go's naming convention

### PR DESCRIPTION
**Pull request by AkihiroSuda**
No substantial code change.
- Api         --> API (suggested by @stevvooe https://github.com/docker/docker/pull/26837#discussion_r80552544)
- Cli         --> CLI
- Http, Https --> HTTP, HTTPS
- Id          --> ID
- Uid,Gid,Pid --> UID,PID,PID
- Ipam      --> IPAM
- Tls         --> TLS (TestDaemonNoTlsCliTlsVerifyWithEnv --> TestDaemonTLSVerifyIssue13964)

Didn't touch in this commit:
- Git: because it is officially "Git": https://git-scm.com/
- Tar: because it is officially "Tar" (in GNU implementation, at least): https://www.gnu.org/software/tar/
- Cpu, Nat, Mac, Ipc, Shm: for keeping a consistency with existing production code (not changable, for compatibility)

Signed-off-by: Akihiro Suda suda.akihiro@lab.ntt.co.jp

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐸

File: integration-cli/docker_api_network_test.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func (s *DockerSuite) TestApiNetworkConnectDisconnect(c *check.C) {
 	deleteNetwork(c, nr.ID, true)
 }
 
-func (s *DockerSuite) TestApiNetworkIpamMultipleBridgeNetworks(c *check.C) {
+func (s *DockerSuite) TestAPINetworkIpamMultipleBridgeNetworks(c *check.C) {
```
Comment (aaronlehmann) [inline integration-cli/docker_api_network_test.go]: Should this be `IPAM`?

Comment (AkihiroSuda) [inline integration-cli/docker_api_network_test.go]: done

**Comment by AkihiroSuda** [comment]
rebased and updated (Ipam->IPAM)

**Comment by stevvooe** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**